rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/robolectric/fragment/CovidJsonFormFragmentTest.java

```diff
@@ -0,0 +1,39 @@
+package io.ona.rdt.robolectric.fragment;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.powermock.reflect.Whitebox;
+
+import io.ona.rdt.fragment.CovidJsonFormFragment;
+import io.ona.rdt.presenter.CovidJsonFormFragmentPresenter;
+import io.ona.rdt.presenter.RDTJsonFormFragmentPresenter;
+import io.ona.rdt.robolectric.RobolectricTest;
+import io.ona.rdt.util.Constants;
+
+public class CovidJsonFormFragmentTest extends RobolectricTest {
+
+    private CovidJsonFormFragment covidJsonFormFragment;
+
+    @Override
+    public void setUp() throws Exception {
+        super.setUp();
+        covidJsonFormFragment = (CovidJsonFormFragment) CovidJsonFormFragment.getFormFragment("step1");
+    }
+
+    @Test
+    public void testFormHasSpecialNavigationRules() throws Exception {
+        Assert.assertTrue(Whitebox.invokeMethod(covidJsonFormFragment, "formHasSpecialNavigationRules", Constants.Encounter.RDT_TEST));
```

Comment:
Could this iterate over all forms in the [set](https://github.com/onaio/rdt-standard/blob/475d773834a237dcbcc2e993a0fdb4da4ad8f86c/rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/fragment/CovidJsonFormFragment.java#L23) instead?

Reply:
done